code/include/observations.h

```diff
@@ -47,14 +47,16 @@ Motivated by investigation into the monocular case of:
 #include "camera_stream.h"
 #include <opencv2/opencv.hpp>
 
+#define TICKS_PER_SEC 1000000000l
+#define TIME_TYPE std::chrono::duration<long int, std::ratio<1l, TICKS_PER_SEC> >
```

Comment:
Can be replaced with 
~~~~~
#define TIME_TYPE std::chrono::steady_clock::duration
~~~~~

If you use `duration_cast` for calling `stretchDistrib`